templates/insights/insights_page.html

```diff
@@ -7,35 +7,48 @@
 {% block content %}
     {% include 'includes/generic-intro.html' %}
     {% include 'includes/hero-img.html' %}
-    {% jumplinks page %}
-
-    {% for block in page.body %}
-        {% include_block block %}
-    {% endfor %}
-
-    {% comment %}
-        NOTE: TO BE REFINED BY FE
-    {% endcomment %}
-    {% if page.similar_items %}
-        <h3>Similar insights</h3>
-        <ol>
-            {% for item in page.similar_items %}
-                <li><a href="{% pageurl item %}">{{ item }}</a></li>
-            {% endfor %}
-        </ol>
-    {% endif %}
-
-    {% comment %}
-        NOTE: TO BE REFINED BY FE
-    {% endcomment %}
-    {% if page.latest_items %}
-        <h3>More insights</h3>
-        <ol>
-            {% for item in page.latest_items %}
-                <li><a href="{% pageurl item %}">{{ item }}</a></li>
+
+    <div class="insights-container">
+        <div class="insights-container__sidebar">
+            {% jumplinks page %}
+        </div>
+
+        <div class="insights-container__main">
+            {% for block in page.body %}
+                {% include_block block %}
             {% endfor %}
-        </ol>
-    {% endif %}
+        </div>
+
+        <div class="insights-container-related-content">
+            <div class="container">
+                {% if page.similar_items %}
+                    <h2 class="insights-container-related-content__heading">Related insights</h3>
+
+                    <ul class="card-group--list-style-none">
+                        {% for item in page.similar_items %}
+                            {% with page=item %}
+                                {% include 'includes/card-group-secondary-nav.html' %}
+                            {% endwith %}
+                        {% endfor %}
+                    </ul>
+                {% endif %}
+
+                {% include 'includes/related-ce-content.html' %}
+
+                {% if page.latest_items %}
+                    <h2 class="insights-container-related-content__heading">More insights</h3>
+
+                    <ul class="card-group--list-style-none">
+                        {% for item in page.latest_items %}
+                            {% with page=item %}
+                                {% include 'includes/card-group-secondary-nav.html' %}
+                            {% endwith %}
```

Comment:
This would be clearer as:
```suggestion
                            {% include 'includes/card-group-secondary-nav.html' with page=item %}
```